src/plugins/home/server/tutorials/haproxy_metrics/index.ts

```diff
@@ -0,0 +1,76 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
```

Comment:
Why change this folder, it's not related to our feature.

Reply:
I deleted this file accidentally in my last PR. So adding it back.